### DISCUSSION TITLE
truthy value should be one of [false, true]

### DISCUSSION THREAD

**Discussion by babafou** [ansible/ansible-lint#2305] [Ideas] [answered]
ansible-lint, by way of yamllint, reports: "truthy value should be one of [false, true]" when using no or yes instead of false or true.

But the documentation of all modules I checked use yes and no. For instance ansible.builtin.user for the append, create_home, force, generate_ssh_key, hidden, local, move_home, non_unique, password_lock. remove and system parameters (which is a lot).

So ansible-lint is not consistent with Ansible documentation. Of course, .ansible-lint makes it possible to silence those warnings but shouldn't it be done by default?

**Comment by konstruktoid** [comment, marked as answer]
Massive work in progress, see e.g https://github.com/ansible/ansible/pull/78429

**Reply by babafou** [reply]
Then it means the documentation is wrong and ansible-lint (well, yamllint) is right.  Fine by me, I'll update my code. Thanks.
